consoles/sshVirtsh.pm

```diff
@@ -242,7 +249,9 @@ sub add_vnc {
     $graphics->setAttribute(autoport    => 'no');
     $graphics->setAttribute(listen      => '0.0.0.0');
     $graphics->setAttribute(sharePolicy => 'force-shared');
-    $graphics->setAttribute(passwd      => $testapi::password);
+    if (my $vnc_password = $testapi::password) {
+        $graphics->setAttribute(passwd => $vnc_password);
+    }
```

Comment:
Likely a blank password is set. Anyway, I'd like to see `passwd` attribute go, we don't use it in qemu.